plugin.video.twitch/addon.xml

```diff
@@ -1,27 +1,30 @@
 <?xml version='1.0' encoding='UTF-8' standalone='yes'?>
-<addon id='plugin.video.twitch' version='1.3.1' name='TwitchTV' provider-name='StateOfTheArt and ccaspers'>
-  <requires>
-    <import addon='xbmc.python' version='2.1.0'/>
-    <import addon='script.module.simplejson' version='2.0.10'/>
-    <import addon="script.module.xbmcswift2" version="1.2.0"/>
-  </requires>
-  <extension point='xbmc.python.pluginsource' library='default.py'>
+<addon id='plugin.video.twitch' version='1.4.0' name='Twitch' provider-name='StateOfTheArt and ccaspers'>
+    <requires>
+        <import addon='xbmc.python' version='2.20.0'/>
```

Comment:
It is intended to support versions as far back as Gotham.
On 11 Apr 2016 3:34 pm, "learningit" <notifications@github.com> wrote:

> In plugin.video.twitch/addon.xml
> <https://github.com/xbmc/repo-plugins/pull/421#discussion_r59214403>:
>
> > @@ -1,27 +1,30 @@
> >  <?xml version='1.0' encoding='UTF-8' standalone='yes'?>
> > -<addon id='plugin.video.twitch' version='1.3.1' name='TwitchTV' provider-name='StateOfTheArt and ccaspers'>
> > -  <requires>
> > -    <import addon='xbmc.python' version='2.1.0'/>
> > -    <import addon='script.module.simplejson' version='2.0.10'/>
> > -    <import addon="script.module.xbmcswift2" version="1.2.0"/>
> > -  </requires>
> > -  <extension point='xbmc.python.pluginsource' library='default.py'>
> > +<addon id='plugin.video.twitch' version='1.4.0' name='Twitch' provider-name='StateOfTheArt and ccaspers'>
> > +    <requires>
> > +        <import addon='xbmc.python' version='2.20.0'/>
>
> You are asking for this PR to be pulled into the "Gotham" repo, but you
> have the 'xbmc.python' version set to Isengard. Which version repo is this
> update intended for?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/xbmc/repo-plugins/pull/421/files/6b1e91bb0e3fdb9ca25bd98adb56ecf9c5fa2dd6#r59214403>
>